internal/db/models/models.go

```diff
@@ -2,7 +2,8 @@ package models
 
 // ListOptions represents pagination and filtering options for list operations
 type ListOptions struct {
```

Comment:
[non blocking] we might need model specific options. Let's just keep this in mind for now.